packages/@aws-cdk/aws-certificatemanager/lib/certificate.ts

```diff
@@ -227,6 +227,11 @@ export class Certificate extends CertificateBase implements ICertificate {
       }
     }
 
+    //check if domain name is less than 64  characters
+    if (props.domainName.length > 64) {
+      throw new Error('Domain name must be less than 64 characters');
```

Comment:
The code indicates that the domain name must be 64 characters or less, not less than 64 characters. Please make sure the code and error message are consistent.

Reply:
Off by one strikes again! Good catch, thanks @TheRealAmazonKendra :)